app/styles/ember-power-select-typeahead.scss

```diff
@@ -1,8 +1,10 @@
+@import "ember-power-select";
```

Comment:
This is the only line that I think should be removed. I haven't documented it tho, but the idea is that the user imports both stylesheets:

```scss
@import "ember-power-select";
@import "ember-power-select-typeahead";
```

and in that order.